config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/validator-labs/validator-plugin-network
-  newTag: latest
+  newName: example.com/validator-plugin-network
```

Comment:
Just checking... do we want "example" here?

Reply:
Thanks, good catch. no...